pyproject.toml

```diff
@@ -69,3 +69,249 @@ docs = {file = ["requirements/optional-docs.txt"]}
 
 [tool.setuptools.packages.find]
 include = ["esmf_regrid*"]
+
+[tool.repo-review]
+ignore = [
+    # https://learn.scientific-python.org/development/guides/style/#PC170
+    "PC170",  # PyGrep hooks
+    # https://learn.scientific-python.org/development/guides/style/#PC180
+    "PC180",  # Uses prettier
+    "PY005",  # Test folder location
+    "GH104",  # Unique names for upload artifact
+    "GH212",  # Require GHA update grouping
+]
```

Comment:
Seems inconsistent to do the link for 2 and not all, your call if you want to remove the links that are there or add the rest.

Reply:
The other equivalent links don't work, I agree it does make it look a bit odd with just two though.